packages/ui/public/locales/en/web_neutron.json

```diff
@@ -0,0 +1,10 @@
+{
+  "params.CCVConsumer": "CCV Consumer",
```

Comment:
This json file was added in a recent commit, but seems only for `en`, I think would be good to add translations (I can see the English keys in the params page, e.g. https://cosmos-nc3xk689i-bigdipper.vercel.app/zht/params )